Mathlib/Algebra/Category/ModuleCat/ChangeOfRings.lean

```diff
@@ -203,6 +203,16 @@ abbrev restrictScalarsComp := restrictScalarsComp'.{v} f g _ rfl
 
 end
 
+noncomputable instance x {R S} [Ring R] [Ring S] (e : R ≃+* S) :
```

Comment:
Should this really be called `x`? :grinning:

Reply:
Good catch! I added a random name in order to #print it.